kubernetes/windows-exporter-daemonset.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app: windows-node-exporter
+  name: windows-node-exporter
+  namespace: monitoring
+spec:
+  selector:
+    matchLabels:
+      app: windows-node-exporter
+  template:
+    metadata:
+      labels:
+        app: windows-node-exporter
+    spec:
+      securityContext:
+        windowsOptions:
+          hostProcess: true
+          runAsUserName: "NT AUTHORITY\\system"
+      hostNetwork: true
+      containers:
+      - args: 
+        - --config.file=%CONTAINER_SANDBOX_MOUNT_POINT%/config.yml
+        name: windows-node-exporter
+        image: jsturtevant/windows-exporter:latest
```

Comment:
could we get this in the same place as node-exporter?

Reply:
yes, I need some guidance from @windows_exporter-reviewers on naming and how to get the image published. I took some of the constructs form node_exporter but not sure if tooling is in place to push the images or if I have the correct docker repository

Reply:
@carlpett @breed808 do you know what needs to be done here?

Reply:
just as an fyi - we don't need windows build machines for this image. We can use buildx running on Linux to build the images. The requirement is that you must have a registry to export too when building since we must use `--output=type=registry` with buildx.